fe/fe-core/src/main/java/com/starrocks/catalog/LocalTablet.java

```diff
@@ -649,4 +673,29 @@ public String getReplicaInfos() {
         }
         return sb.toString();
     }
+
+    // Note: this method does not require db lock to be held
+    public boolean quorumReachVersion(long version, long quorum, TxnFinishState finishState) {
+        // TODO(cbl): support tablets doing schemachange/rollup
+        long valid = 0;
+        synchronized (replicas) {
+            for (Replica replica : replicas) {
+                long replicaId = replica.getId();
+                long replicaVersion = replica.getVersion();
+                if (replicaVersion > version) {
```

Comment:
Why replicaVersion > version, valid++?

Reply:
causes it is `valid`?  replicaVersion >= version means version is up to date